synapse/lib/storm.lark

```diff
@@ -166,10 +169,14 @@ CMPR: /(?!<-)[@?!<>^~=][@!<>^~=]*/
 
 _safe_cmpr: BYNAME | CMPR
 
-_valu: NONQUOTEWORD | valulist | _varvalu | relpropvalu | univpropvalu | tagvalu | tagpropvalu
+_basevalu.1: valulist | _varvalu | relpropvalu | univpropvalu | tagvalu | tagpropvalu
     | DOUBLEQUOTEDSTRING | SINGLEQUOTEDSTRING | dollarexpr | embedquery
 
-valulist: "(" [_WS? _valu (_WS? "," _WS? _valu)*] _WS? ["," _WS?] ")"
+_valu: _basevalu | NONQUOTEWORD
+_toknvalu: _basevalu | WORDTOKN
+_listvalu: _basevalu | LISTTOKN
```

Comment:
"Your scientists were so preoccupied with whether or not they could, they didn’t stop to think if they should."

Regular users have to understand this stuff.